config/database.yml.example

```diff
@@ -10,8 +10,8 @@ development:
 
 production:
   <<: *default
-  username: mapknitter
-  password: mapknitter
+  username: root
```

Comment:
Why was this change needed? I don't think we should change this since we don't know the credentials of the production env.

Reply:
Authentication problem
https://travis-ci.org/kaustubh-nair/mapknitter/builds/545242931#L767

Reply:
We'll have to create a mapknitter user before we can use it.
It's anyway an example config file, I doubt it would affect the actual `database.yml` on production.
What do you think @jywarren ?